apps/swirl-docs/src/pages/api/tokens/v0/[tokengroup].ts

```diff
@@ -0,0 +1,134 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+import type { NextApiRequest, NextApiResponse } from "next";
+import {
+  cssDark,
+  cssLight,
+  flutterDark,
+  flutterLight,
+  Formats,
+  lessDark,
+  lessLight,
+  scssDark,
+  scssLight,
+  TokenFormats,
+  TokenGroupType,
+  tokenMap,
+} from "./utils";
+const swirlTailwindJson = require("@getflip/swirl-tokens/dist/tailwind/swirl-tailwind.json");
+
+function getTokensForType(type: TokenGroupType): Object {
+  const tokenGroup = tokenMap.get(type);
+  let tokens: any = {};
+
+  if (!tokenGroup) {
+    return {};
+  }
+
+  if (type === "all") {
+    return swirlTailwindJson;
+  }
+
+  tokenGroup.forEach((tokenType) => {
+    tokens = {
+      ...tokens,
+      ...swirlTailwindJson[tokenType],
+    };
+  });
+
+  return tokens;
+}
+
+export default function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<Buffer | any>
+) {
+  const { tokengroup, format, scheme } = req.query;
+
+  const hasNoFormatOrIsJson = !format || format === "json";
+
+  if (hasNoFormatOrIsJson) {
+    jsonResponse(res, tokengroup);
+  }
+
+  if (!Formats.includes(format as string)) {
+    res.status(404).json({
+      error: "There is no format corresponding your request",
+    });
+  } else {
+    switch (format) {
+      case TokenFormats.CSS:
+        res.setHeader("Content-Type", "text/css");
+        if (scheme === "dark") {
+          res.status(200).send(cssDark);
+        } else {
+          res.status(200).send(cssLight);
+        }
+        break;
+      case TokenFormats.FLUTTER:
+        res.setHeader("Content-Type", "text/dart");
```

Comment:
**question:** Are text/dart, text/less and text/scss accepted mime types? If not, I think text/plain or application/octet-stream would be the safest option.